src/controllers/bookController.js

```diff
@@ -0,0 +1,18 @@
+const  bookService  = require("../services/bookService");
+const { catchAsync } = require("../utils/error");
+
+const getAllBooks = catchAsync(async (req, res) => {
+  const books = await bookService.getAllBooks();
+  res.status(200).json(books);
+});
+
+const getBooksByMain = catchAsync(async (req, res) => {
+  const id = req.params.id;
+  const books = await bookService.getBooksByMain(id);
+  res.status(200).json(books);
+});
```

Comment:
함수명이 조금 불명확한 것 같습니다.
book의 id 값으로 특정 값을 조회하는 것이라면 다음과 같이 수정하면 좋을 것 같습니다.

```suggestion
const getBooksById = catchAsync(async (req, res) => {
  const bookId = req.params.id;
  const books = await bookService.getBooksByMain(bookId);
  return res.status(200).json(books);
});
```